src/Stratis.Bitcoin/Consensus/ConsensusRuleEngine.cs

```diff
@@ -19,7 +19,7 @@ public abstract class ConsensusRuleEngine : IConsensusRuleEngine
         private readonly ILoggerFactory loggerFactory;
 
         /// <summary>Instance logger.</summary>
-        private readonly ILogger logger;
+        protected readonly ILogger logger;
```

Comment:
why

Reply:
To be accessed from the inherited classes